### ISSUE TITLE
Users count is resetting on reset

### ISSUE THREAD

**Issue by jeongm-in** [jeongm-in/Disciple-of-vow#5]

**Comment by Contrastellar** [comment]
When is the bug occurring? I just tried mucking around with it on two separate Firefox 98.0 /w Windows Version 10.0.19044 Build 19044.

**Comment by Contrastellar** [comment]
I'm noticing that the database isn't showing users disconnecting correctly when I close a tab. Is the DB automatically "closing" connections?

**Comment by jeongm-in** [comment]
Yes, something is off with how we keep track of users connected to it. In general this shouldn't be a matter as the number's there to make sure everyone in the fireteam is connected to the session. I think we would need a complete rewrite for when/how we increment/decrement the number. We do try to decrement the  number when someone closes the tab [here](https://github.com/jeongm-in/Disciple-of-vow-toolkit/blob/dfe2d5c3fcf650719497f0f1e9222db3ea47bfb2/public/scripts/acquisition.js#L124), so the number on the DB should decrement by 1 whenever people leave. However I confirmed the number doesn't actually decrease on the DB side, so that block inside `onBeforeUnload` event is where we might want to look into. 

![image](https://user-images.githubusercontent.com/8725873/158078434-2096795b-2323-4274-861c-94dbf14b5453.png)
****
1. Logged into session `asdf` on Desktop
2. Logged into session `asdf` on mobile
3. Confirmed the number updates on both sessions whenever new connection is made 
4. Confirmed the DB `uConnected` field doesn't decrement when closing the tab
